src/style.css

```diff
@@ -1,3 +1,87 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  width: 1200px;
+  background: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  flex-shrink: 0;
+}
+
+@media (min-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+@media (min-width: 1200px) {
+  .header {
+    width: 1200px;
+  }
+
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  align-items: center;
+}
+
+.logo__img {
+  flex-shrink: 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  position: relative;
+
+}
+
+.nav__list li:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  color: #000;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  text-decoration: none;
+}
+
+.nav__link:hover,
+.nav__link:active,
+.is-active {
+  color: #00acdc;
```

Comment:
- it's a good practice to move all colors to the variable and reuse them
 - please also move other duplicated styles, for example, margins, paddings, font-size, etc.